docs/x509.rst

```diff
@@ -652,6 +652,32 @@ X.509 Extensions
     purposes indicated in the key usage extension. The object is
     iterable to obtain the list of :ref:`extended key usage OIDs <eku_oids>`.
 
+.. class:: AuthorityKeyIdentifier
+
+    .. versionadded:: 0.9
+
+    The authority key identifier extension provides a means of identifying the
+    public key corresponding to the private key used to sign a certificate.
+
+    .. attribute:: key_identifier
+
+        :type: bytes
+
+        A value derived from the public key used to verify the certificate's
+        signature.
```

Comment:
The spec is noncommittal on what that means, unfortunately. There's a "standard" way that is pretty much what everybody does (to my knowledge) but technically you can put whatever the hell you want here. RFC 5280 is not great sometimes.